sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -3207,6 +3209,16 @@ public void setSpanFactory(final @NotNull ISpanFactory spanFactory) {
     this.spanFactory = spanFactory;
   }
 
+  @ApiStatus.Experimental
```

Comment:
do we still want to keep these marked as experimental with the annotation or can we remove it, too?

Reply:
We could remove them or leave them in to give us a bit of freedom to fix any bugs in the API more easily.

Reply:
it's fine to keep it, just ensuring it was intentional